packages/Overlay/src/Overlay.js

```diff
@@ -93,7 +99,7 @@ const Overlay = props => {
   return (
     <>
       {isOpen && <LockBodyScroll />}
-      <Portal>
+      <Portal container={container}>
```

Comment:
nit: I think root would be a more descriptive prop name, but I'm not too strong about this.